mmn/node_information.go

```diff
@@ -64,6 +66,7 @@ func CollectNodeData(client *client, resolver ip.Resolver) func(string) {
 		info.MACAddress = hex.EncodeToString(sha256Bytes[:])
 		info.LocalIP = outboundIp
 		info.Identity = identity
+		info.VendorID = config.Current.GetString(config.VendorIDFlag.Name)
```

Comment:
Depending on global state makes code harder to test. I would suggest have app Config struct which can be passed via params.

```go
func MyFunc(cfg config.Config, ...) {
    cfg.GetString(config.VendorIDFlag.Name)
}
```